test/e2e/tests/account-details.spec.js

```diff
@@ -110,6 +110,10 @@ describe('Show account details', function () {
         );
         await driver.fill('[placeholder="Account 2"]', '2nd account');
         await driver.clickElement({ text: tEn('create'), tag: 'button' });
+        await driver.waitForElementNotPresent({
+          text: tEn('create'),
+          tag: 'button',
+        });
```

Comment:
Creating an account triggers a vault update, which now requires more time and needs to be awaited somehow